johnslist/dbtest/models.py

```diff
@@ -88,19 +89,29 @@ class Job(models.Model):
     def __unicode__(self):
         return self.name
     name = models.CharField('Job Name',max_length=128)
-    client_organization = models.CharField('What organization do you represent?', max_length=64) #what entity/organization needs this job?
-    description = models.TextField('Job Description', max_length=256) #short description
-    deliverable = models.TextField('Deliverable', max_length=256) #end product to be delivered
-    duedate = models.DateTimeField('Date Due') #when Job is due for completion
-    stakeholders = models.TextField('Stakeholders') #all persons who may be affected by project
-    additional_information = models.TextField('Additional Information', blank = True) #important technical requirements
-    budget = models.CharField('Budget', max_length=64) #budget estimate
-    attachments = models.FileField(upload_to='job', blank = True) #file attachments
-    creator = models.ForeignKey(User,related_name = 'jobs')  # User -o= Job
-    organization = models.ManyToManyField(Organization, through = 'JobRequest')
+    # what entity/organization needs this job?
+    client_organization = models.CharField('What organization do you represent?', max_length=64)
+    # short description
+    description = models.TextField('Job Description', max_length=256)
+    # end product to be delivered
+    deliverable = models.TextField('Deliverable', max_length=256)
+    # when Job is due for completion
+    duedate = models.DateTimeField('Date Due')
+    # all persons who may be affected by project
+    stakeholders = models.TextField('Stakeholders')
+    # important technical requirements
+    additional_information = models.TextField('Additional Information', blank = True)
+    # budget estimate
+    budget = models.CharField('Budget', max_length=64)
+    # file attachments
+    attachments = models.FileField(upload_to='job', blank = True)
+    creator = models.ForeignKey(User,related_name = 'jobs')
+    organizations = models.ManyToManyField(Organization, through = 'JobRequest')
     contact_information = models.CharField('Contact Information', max_length = 256, blank = True)
-    closed = models.NullBooleanField(default = False)  # Job is closed after a jr is confirmed
-    # categories = models.ManyToManyField(Category) #some tags to determine what organizations to submit job to
+    #  Job is closed after a jr is confirmed
+    closed = models.NullBooleanField(default = False)
```

Comment:
i updated "closed" attribute to BooleanField in develop. We talked about this few weeks ago. Just a FYI